isubit/luggage_frontpanel_block
# Lugg 1045

Adds a text size field for Cards. 

To Test: 
1. Create a card.
2. Is there a text size field?
3. Is it set by default to Medium Text?
4. When you change the text to small or large, does it change as you expect?
